tests/tools/onert_train/src/onert_train.cc

```diff
@@ -138,11 +138,16 @@ int main(const int argc, char **argv)
     NNPR_ENSURE_STATUS(nnfw_train_get_traininfo(session, &tri));
 
     // overwrite training information using the arguments
-    tri.batch_size = args.getBatchSize();
-    tri.learning_rate = args.getLearningRate();
-    tri.loss_info.loss = args.getLossType();
-    tri.loss_info.reduction_type = args.getLossReductionType();
-    tri.opt = args.getOptimizerType();
+    if (tri.batch_size == 0 && !args.getBatchSize().has_value())
+      tri.batch_size = 1;
+    else
+      tri.batch_size = args.getBatchSize().value_or(tri.batch_size);
+
+    tri.learning_rate = args.getLearningRate().value_or(tri.learning_rate);
+    tri.loss_info.loss = args.getLossType().value_or(tri.loss_info.loss);
+    tri.loss_info.reduction_type =
+      args.getLossReductionType().value_or(tri.loss_info.reduction_type);
+    tri.opt = args.getOptimizerType().value_or(tri.opt);
```

Comment:
Okay, after this line `tri`'s values are filled sth(it could be valid or invalid.) If these values are invalid, nnfw_api throws error.